### PR TITLE
Minor code quality improvements based on LGTM

### PR DESCRIPTION
**Pull request by AnnikaCodes**
Basically I went through the warnings at https://lgtm.com/projects/g/smogon/pokemon-showdown/alerts (which is linked in our README.md), and fixed the ones that weren't extraneous.

### CODE REVIEW

File: data/mods/mixandmega7/scripts.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const Scripts: ModdedBattleScriptsData = {
 		} = {
 			abilities: megaSpecies.abilities,
 			baseStats: {},
-			weighthg: megaSpecies.weighthg - megaSpecies.weighthg,
+			weighthg: 0,
```
Comment (Slayer95) [inline data/mods/mixandmega7/scripts.ts:80]: ``megaSpecies.weighthg - baseSpecies.weighthg``

File: server/chat-commands/info.ts [hunk 2]
```diff
@@ -773,12 +773,10 @@ export const commands: ChatCommands = {
 				throw new Error(`Unrecognized searchType`);
 			}
 
-			if (details) {
```
Comment (Slayer95) [inline server/chat-commands/info.ts:776]: ``if (showDetails)``

File: server/tournaments/index.ts [hunk 3]
```diff
@@ -756,7 +756,7 @@ export class Tournament extends Rooms.RoomGame {
 		if (user) {
 			user.sendTo(this.room, '|tournament|update|{"isJoined":false}');
 			if (reason) {
-				user.popup(`|modal|You have been disqualified from the tournament in ${this.room.title + (reason ? ':\n\n' + reason : '.')}`);
+				user.popup(`|modal|You have been disqualified from the tournament in ${this.room.title}:\n\n${reason}.`);
```
Comment (KrisXV) [inline server/tournaments/index.ts:759]: I would just remove the `if` statement:

```js
user.popup(`|modal|You have been disqualified from the tournament in ${this.room.title}${reason ? `:\n\n${reason}` : `.`}`);
```

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**